src/OpenSage.Game/Data/InstallationLocator.cs

```diff
@@ -67,10 +67,17 @@ public class EnvironmentInstallationLocator : IInstallationLocator
     {
         public IEnumerable<GameInstallation> FindInstallations(IGameDefinition game)
         {
-            String path = Environment.GetEnvironmentVariable(game.Identifier.ToUpperInvariant() + "_PATH");
+            var identifier = game.Identifier.ToUpperInvariant() + "_PATH";
+            var path = Environment.GetEnvironmentVariable(identifier);
 
-            if (path == null)            
+            if (path == null)
+            {
+                path = Environment.GetEnvironmentVariable(identifier, EnvironmentVariableTarget.User);
```

Comment:
Interesting - does this find values that the method above doesn't? I thought on Windows the current process inherited the user's environment variables?

Reply:
(because `Environment.GetEnvironmentVariable(string)` is the same as calling `Environment.GetEnvironmentVariable(string, EnvironmentVariableTarget.Process)`.)

Reply:
Yea, the user path variables are not found by the current implementation.

Reply:
Okay, looks like a good change then.